incubator/hnc/e2e/tests/delete_crd_test.go

```diff
@@ -62,4 +63,21 @@ var _ = Describe("Delete-anchor-crd", func() {
 		// verify
 		mustRun("kubectl get -oyaml rolebinding foo -n", nsChild)
 	})
+
+	It("should delete the HC CRD, anchor CRD, and verify that the HNC CRDs are gone", func() {
```

Comment:
I think this description should be `should fully delete all CRDs`

Reply:
done